main.go

```diff
@@ -12,17 +12,26 @@ import (
 	"github.com/bobuhiro11/gokvm/kvm"
 	"github.com/bobuhiro11/gokvm/machine"
 	"github.com/bobuhiro11/gokvm/term"
+	"github.com/bobuhiro11/gokvm/tools"
 )
 
 func main() {
 	// This line break is required by golangci-lint but
 	// such breaks are considered an anti-pattern
 	// at Google.
-	c, err := flag.ParseArgs(os.Args)
+	c, debug, err := flag.ParseArgs(os.Args)
 	if err != nil {
 		log.Fatalf("ParseArgs: %v", err)
 	}
 
+	if *debug {
+		if err := tools.TestCaps(); err != nil {
+			log.Fatalf("TestCaps: %v", err)
+		}
+
+		os.Exit(0)
```

Comment:
Cloud you please remove this line?
I think it would be good to boot virtual machine while outputting the list of capabilities as debug logs.

Reply:
done